test/data/bad/excessive_precision.geojson

```diff
@@ -0,0 +1 @@
+{ "type": "Point", "coordinates": [100.0000000001, 5.0000000001] }
```

Comment:
When I wrote that, it was in the context of geojsonhint showing an error. So IMO 6-8 decimal places is perfectly reasonable for some survey applications and I didn't want to yell too loudly about it.

But we need to think about differentiating between warnings vs errors. If it's just a warning, sure let's drop the precision to 6.